ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -233,4 +233,22 @@ message DisclosedContract {
   // The contract metadata from the create event.
   // Required
   ContractMetadata metadata = 4;
+
+  // The signatories for this contract as specified by the template.
+  // Required
```

Comment:
@tudor-da @mziolekda : this is a breaking change, which might be fine to do, but definitely must be documented in the release notes.

FYI: @isegall-da @cocreature -- once we consume a Canton bump with this change, in case it actually lands, we'll need to adapt our command submissions to also produce the signatories. We'll probably just want to compute them in Scan for the contracts which we disclose.